### DISCUSSION TITLE
Build for App Store with ti cli

### DISCUSSION THREAD

**Discussion by caspahouzer** [appcelerator/titanium_mobile#12556] [Q&A] [answered]
I switched from appc to ti cli to avoid login to the platform.

Now on building my app for the App Store, the cli infos me
```
[ERROR] This application cannot be built with the Titanium open source SDK because it is an Appcelerator Platform registered application. Please use the Appcelerator Platform CLI tools or Appcelerator Studio to build this application.
```
This is my build exec (Copied and modified from vsCode)
```
ti build -f --platform ios --project-dir ~/Documents/Apps/Systemhaus --deploy-type production --distribution-name "Apple Distribution: SYNAXON AG (3BY9MGDNUT)" --keychain --pp-uuid 3f9504e9-5f12-4103-8bf4-87e62a09ad28 --target dist-appstore --skip-js-minify --no-progress-bars --no-prompt --no-banner --prompt-type socket-bundle --prompt-port 53711 --no-banner
```

**Comment by m1ga** [comment, marked as answer]
generate a new GUID https://www.guidgenerator.com/ and remove the appc-app-id

**Reply by caspahouzer** [reply]
@m1ga reliable and helpful as always. Thank you

**Reply by m1ga** [reply]
Would be a good note for a transition guide if you want to do it now already. I'll add a small section to my `from zero to app` tutorials!

Later I'm sure it won't matter since there won't be any server to check that ID against :smile:

**Reply by caspahouzer** [reply]
Yes, I would like to free myself from all axway obligations as soon as possible ;-)

**Reply by jmcerrejon** [reply]
Put my 2 cents with a cli command:

```sh
sed -i -e "s/\(<guid>\).*\(<\/guid>\)/<guid>$(uuidgen)<\/guid>/g" tiapp.xml
```

Yes, you can generate random _uuid_ on a terminal with the command `uuidgen`

**Reply by caspahouzer** [reply]
@jmcerrejon that's pretty cool 👍
